tests/docker/docker-compose.yml

```diff
@@ -14,10 +14,14 @@ services:
       KAMON_ENABLED: "false"
       LOG_LEVEL: "INFO"
     image: bluebrain/nexus-delta:latest
-    entrypoint: [ "bin/wait-for-it.sh", "-s", "-t", "0", "postgres:5432", "--",
-                  "bin/wait-for-it.sh", "-s", "-t", "0", "elasticsearch:9200", "--",
-                  "./bin/delta-app",
-                  "-Xmx512m" ]
+    entrypoint:
+      - '/bin/bash'
+      - '-c'
+      - |
+        ln -s /opt/docker/plugins/disabled/project-deletion.jar /opt/docker/plugins/project-deletion.jar &&
```

Comment:
The project deletion plugin is bundled in a different path to avoid enabling it by mistake so this extra step is needed to enable it